project.clj

```diff
@@ -20,7 +20,7 @@
                  [commons-io]
                  [io.dropwizard.metrics/metrics-core]
 
-                 [puppetlabs/ssl-utils]
+                 [puppetlabs/ssl-utils "2.0.1-SNAPSHOT"]
```

Comment:
We've release 3.0.1 with the needed changes in it. I suppose we should do the clj-parent dance so we can just remove the explicit dependency here.